File: JS/URI/1036.js
var input = require('fs').readFileSync('stdin', 'utf8');

var lines = input.split(' ');

var a = parseFloat(lines.shift());

var b = parseFloat(lines.shift());

var c = parseFloat(lines.shift());

// es b x b, no se porque no coje el signo pero si lo agregas en tu codigo seguro servira

var pi = (b * b) - (4 * a * c);

if(pi<0){

console.log('Impossivel calcular');

} else if (a === 0){

console.log('Impossivel calcular');

} else{

var r1 = parseFloat((-b + Math.sqrt(pi)) / (a + a));

var r2 = parseFloat((-b - Math.sqrt(pi)) / (a + a));

console.log('R1 = ' + r1.toFixed(5));

console.log('R2 = ' + r2.toFixed(5));

}

File: Python/URI/1036.py
import math 
x = input().split(' ');

a = float(x[0]) 
b = float(x[1]) 
c = float(x[2])

delta = (b*b)-(4*a*c)

if (float(delta) < 0 or float(a) == 0):
    print("Impossivel calcular")
else: 
    r1 = (-b + math.sqrt(delta))/(2*a)
    r2 = (-b - math.sqrt(delta))/(2*a)
    print("R1 = {:.5f}".format(r1))
    print("R2 = {:.5f}".format(r2))
